src/utils/index.ts

```diff
@@ -0,0 +1,17 @@
+import type { IComputePositions } from './compute-tooltip-position-types'
+import computeTooltipPosition from './compute-tooltip-position'
+import cssSupports from './css-supports'
+import cssTimeToMs from './css-time-to-ms'
+import debounce from './debounce'
+import getScrollParent from './get-scroll-parent'
+import useIsomorphicLayoutEffect from './use-isomorphic-layout-effect'
+
+export type { IComputePositions }
+export {
+  computeTooltipPosition,
+  cssSupports,
+  cssTimeToMs,
+  debounce,
+  getScrollParent,
+  useIsomorphicLayoutEffect,
+}
```

Comment:
We had some inconsistent exports here. All util functions are now default exported, and available with named export from `utils/index.ts`.